src/earn/EarnHome.tsx

```diff
@@ -0,0 +1,396 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import { default as React, useMemo, useRef, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { LayoutChangeEvent, StyleSheet, Text, View } from 'react-native'
+import { ScrollView } from 'react-native-gesture-handler'
+import Animated, {
+  interpolateColor,
+  useAnimatedScrollHandler,
+  useAnimatedStyle,
+  useSharedValue,
+} from 'react-native-reanimated'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import BottomSheet, { BottomSheetRefType } from 'src/components/BottomSheet'
+import FilterChipsCarousel, {
+  FilterChip,
+  NetworkFilterChip,
+  TokenSelectFilterChip,
+  isNetworkChip,
+  isTokenSelectChip,
+} from 'src/components/FilterChipsCarousel'
+import TokenBottomSheet, { TokenPickerOrigin } from 'src/components/TokenBottomSheet'
+import NetworkMultiSelectBottomSheet from 'src/components/multiSelect/NetworkMultiSelectBottomSheet'
+import EarnTabBar from 'src/earn/EarnTabBar'
+import PoolList from 'src/earn/PoolList'
+import { EarnTabType, Pool } from 'src/earn/types'
+import { Screens } from 'src/navigator/Screens'
+import useScrollAwareHeader from 'src/navigator/ScrollAwareHeader'
+import { StackParamList } from 'src/navigator/types'
+import { useSelector } from 'src/redux/hooks'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Shadow, Spacing, getShadowStyle } from 'src/styles/styles'
+import { tokensByIdSelector } from 'src/tokens/selectors'
+import { TokenBalance } from 'src/tokens/slice'
+import { NetworkId } from 'src/transactions/types'
+
+const HEADER_OPACITY_ANIMATION_START_OFFSET = 44
+const HEADER_OPACITY_ANIMATION_DISTANCE = 20
+
+type Props = NativeStackScreenProps<StackParamList, Screens.EarnHome>
+
+// TODO (ACT-1268): Replace with getEarnPositions
+const pools: Pool[] = [
+  {
+    poolId: 'aArbUSDCn',
+    networkId: NetworkId['arbitrum-one'],
+    tokens: [`${NetworkId['arbitrum-one']}:0xaf88d065e77c8cc2239327c5edb3a432268e5831`],
```

Comment:
do you need this if deposit and pool token ids are separate entries?

Reply:
It's fine, it'll be part of the position anyway. I'll update this as I start working on the task to connect it to getEarnPositions.